bare-metal-fd-ts/index.ts

```diff
@@ -0,0 +1,62 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as svmkit from "@svmkit/pulumi-svmkit";
+
+const solanaConfig = new pulumi.Config("solana");
+const remote = new pulumi.Config("remote");
+
+// Lookup information about the Solana network.
+const networkName =
+  solanaConfig.get<svmkit.solana.NetworkName>("network") ??
+  svmkit.solana.NetworkName.Testnet;
+const networkInfo = svmkit.networkinfo.getNetworkInfoOutput({ networkName });
+
+// Create some keys for this validator to use.
+export const validatorKey = new svmkit.KeyPair("validator-key").json;
+export const voteAccountKey = new svmkit.KeyPair("vote-account-key").json;
+export const withdrawerKey = new svmkit.KeyPair("withdrawer-key").json;
+
+// Point pulumi-svmkit at the AWS EC2 instance's SSH connection.
```

Comment:
EC2 would be VM, yes? Our did we get a baremetal ec2?

Reply:
Oops, you caught me in a copy-paste typo.  Lemme fix that.